src/main/update/launcher-update.ts

```diff
@@ -141,6 +111,36 @@ export async function update(update: Update, listeners: IUpdateOptions) {
   console.log("launcherDownloadUrl: ", launcherDownloadUrl);
   console.log("playerDownloadUrl: ", playerDownloadUrl);
 
+  if (peerVersionNumber <= localVersionNumber) {
```

Comment:
Could you comment on the reason for this change? I can't quite grasp what difference moving this chunk make.

Reply:
just... for creating `playerDownloadUrl` before creating lockfile

Reply:
Ooh I see.